frontend/src/app/app.module.ts

```diff
@@ -7,20 +7,24 @@ import { AppComponent } from './app.component';
 import { HomeComponent } from './pages/home/home.component';
 import { HeaderComponent } from './features/header/header.component';
 
+import { MatButtonModule } from '@angular/material/button'
 import { MatSlideToggleModule } from '@angular/material/slide-toggle';
+import { AssistanceSelectionComponent } from './features/assistance-selection/assistance-selection.component';
```

Comment:
These should be listed alphabetically my what's imported, grouped by type, for readability. `AssistanceSelectionComponent` should be moved above the `HomeComponent`.